src/api/spec/db/data/remove_duplicate_repositories_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+require Rails.root.join('db/data/20170306084550_remove_duplicate_repositories.rb')
+require Rails.root.join('db/migrate/20170306084558_change_repositories_remote_project_name_to_not_null.rb')
+
+RSpec.describe RemoveDuplicateRepositories, type: :migration do
+  # This migration does not allow NULL values for remote_project_name column
+  # We need this migration to not be run to create test data
+  # 20170306084558_change_repositories_remote_project_name_to_not_null.rb
+  let(:schema_migration_version) { 20_170_306_084_558 }
+  let(:schema_migration) { ChangeRepositoriesRemoteProjectNameToNotNull.new }
+  let(:data_migration_version) { 20_170_306_084_550 }
+  let(:data_migration) { RemoveDuplicateRepositories.new }
+
+  describe '.up' do
+    before do
+      schema_migration.down
+    end
+
+    after do
+      DataMigrate::DataSchemaMigration.find_or_create_by(version: data_migration_version.to_s)
```

Comment:
I would like `data_migration_version` not to be a variable. One less indirection to track.